src/sentry/release_health/metrics.py

```diff
@@ -606,3 +601,342 @@ def extract_row_info(row: Mapping[str, Union[OrganizationId, str]]) -> ReleaseNa
+                reverse_tag_value(org_id, row[session_status_column_name]),
+            )
+            rv_sessions[key] = row["value"]
+
+        return rv_sessions
+
+    @staticmethod
+    def _get_users_and_crashed_users_for_overview(
+        where: List[Union[BooleanCondition, Condition]], org_id: int
+    ) -> Mapping[Tuple[int, str, str], int]:
+        release_column_name = tag_key(org_id, "release")
+        session_status_column_name = tag_key(org_id, "session.status")
+
+        aggregates: List[SelectableExpression] = [
+            Column(release_column_name),
+            Column("project_id"),
+            Column(session_status_column_name),
+        ]
+
+        # Count of users and crashed users
+        rv_users: Mapping[Tuple[int, str, str], int] = {}
+
+        for row in raw_snql_query(
+            Query(
+                dataset=Dataset.Metrics.value,
+                match=Entity("metrics_sets"),
+                select=aggregates + [Column("value")],
+                where=where
+                + [
+                    Condition(Column("metric_id"), Op.EQ, metric_id(org_id, "user")),
+                    Condition(
+                        Column(session_status_column_name),
+                        Op.IN,
+                        get_tag_values_list(org_id, ["crashed", "init"]),
+                    ),
+                ],
+                groupby=aggregates,
+            ),
+            referrer="release_health.metrics.get_users_and_crashed_users_for_overview",
+        )["data"]:
+            key = (
+                row["project_id"],
+                reverse_tag_value(org_id, row[release_column_name]),
+                reverse_tag_value(org_id, row[session_status_column_name]),
+            )
+            rv_users[key] = row["value"]
+
+        return rv_users
+
+    @staticmethod
+    def _get_health_stats_for_overview(
+        where: List[Union[BooleanCondition, Condition]],
+        org_id: int,
+        health_stats_period: StatsPeriod,
+        stat: OverviewStat,
+        now: datetime,
+    ):
+        release_column_name = tag_key(org_id, "release")
+        session_status_column_name = tag_key(org_id, "session.status")
+        session_init_tag_value = tag_value(org_id, "init")
+
+        stats_rollup, stats_start, stats_buckets = get_rollup_starts_and_buckets(
+            health_stats_period
+        )
+
+        aggregates: List[SelectableExpression] = [
+            Column(release_column_name),
+            Column("project_id"),
+            Column("bucketed_time"),
+        ]
+
+        rv = defaultdict(lambda: _make_stats(stats_start, stats_rollup, stats_buckets))
+
+        for row in raw_snql_query(
+            Query(
+                dataset=Dataset.Metrics.value,
+                match=Entity(
+                    {
+                        "users": "metrics_sets",
+                        "sessions": "metrics_counters",
+                    }[stat]
+                ),
```

Comment:
I would move this dict outside of the query in favor of readability
```
{
    "users": "metrics_sets",
     "sessions": "metrics_counters",
}[stat]
```

Reply:
Especially that is reused again in line 799

Reply:
the dict on 799 has different values though

Reply:
Oh yeah you are right.. I would still move them both to be honest and just create an internal function that returns the correct values.. its kinda hard to read

Reply:
yeah I moved both